stable/traefik/templates/configmap.yaml

```diff
@@ -68,6 +68,9 @@ data:
     storage = "/acme/acme.json"
     entryPoint = "https"
     onHostRule = true
+    {{- if eq .Values.acme.challengeType "dns-01" }}
+    dnsProvider = "{{ .Values.acme.dnsProvider.name }}"
```

Comment:
It seems that `dnsProvider` is deprecated, per Traefik's docs:

```
# Use a DNS-01 acme challenge rather than TLS-SNI-01 challenge
#
# Optional (Deprecated, replaced by [acme.dnsChallenge])
#
# dnsProvider = "digitalocean"
```

```
# Use a DNS-01 acme challenge rather than TLS-SNI-01 challenge
#
# Optional
#
# [acme.dnsChallenge]

  # Provider used.
  #
  # Required
  #
  # provider = "digitalocean"

  # By default, the provider will verify the TXT DNS challenge record before letting ACME verify.
  # If delayBeforeCheck is greater than zero, avoid this & instead just wait so many seconds.
  # Useful if internal networks block external DNS queries.
  #
  # Optional
  # Default: 0
  #
  # delayBeforeCheck = 0
```